### PR TITLE
TagBox: add Vue and React code snippets

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: concepts/05 Widgets/TagBox/02 Data Binding/05 Simple Array/05 Array Only.md [hunk 1]
```diff
@@ -134,12 +247,82 @@ You can create a [Query](/concepts/70%20Data%20Binding/5%20Data%20Layer/6%20Quer
...
+    import query from "devextreme/data/query";
+
+    export default {
+        components: {
+            DxTagBox
+        },
+        data() {
+            return {
+                products: [
+                    { name: 'HD Video Player', price: 100 },
```
Comment (ksercs) [inline concepts/05 Widgets/TagBox/02 Data Binding/05 Simple Array/05 Array Only.md:277]: Shouldn't we add some more items to demonstrate the sort?

File: concepts/05 Widgets/TagBox/07 Grouping/05 Customize Group Headers.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 By default, group headers display text of the **key** field in a bold font. You can define a custom template for the group headers if you need to. For AngularJS and Knockout apps, DevExtreme provides the [dxTemplate](/api-reference/10%20UI%20Widgets/Markup%20Components/dxTemplate '/Documentation/ApiReference/UI_Widgets/Markup_Components/dxTemplate/') markup component. The following code shows how to use **dxTemplate** to define a template for group headers.
```
Comment (ksercs) [inline concepts/05 Widgets/TagBox/07 Grouping/05 Customize Group Headers.md]: Please, help to rework this topic

File: concepts/05 Widgets/TagBox/02 Data Binding/05 Simple Array/05 Array Only.md (hunk 1 above)
Comment (RomanTsukanov) [reply concepts/05 Widgets/TagBox/02 Data Binding/05 Simple Array/05 Array Only.md:277]: Agreed

File: concepts/05 Widgets/TagBox/02 Data Binding/05 Simple Array/10 ArrayStore.md [hunk 3]
```diff
@@ -98,12 +177,82 @@ Data kept in the **ArrayStore** can be processed in a [DataSource](/api-referenc
...
+                store: products,
+                sort: { getter: "name", asc: true }
+            })
+        }
+
+        render() {
+            return (
+                <TagBox
+                    dataSource={this.productsDataSource}
+                    displayExpr="name"
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/TagBox/02 Data Binding/05 Simple Array/10 ArrayStore.md:249]: valueExpr?

File: concepts/05 Widgets/TagBox/07 Grouping/05 Customize Group Headers.md (hunk 2 above)
Comment (RomanTsukanov) [reply concepts/05 Widgets/TagBox/07 Grouping/05 Customize Group Headers.md]:
```suggestion
To customize group headers, specify a [groupTemplate](/Documentation/ApiReference/UI_Widgets/dxTagBox/Configuration/#groupTemplate). In Angular and Vue, you can declare it in the markup. In React, you can use a rendering function (shown in the code below) or component. If the template is unspecified, group headers display the **key** data field in bold font.
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/TagBox/02 Data Binding/05 Simple Array/05 Array Only.md (hunk 1 above)
Comment (ksercs) [reply concepts/05 Widgets/TagBox/02 Data Binding/05 Simple Array/05 Array Only.md:277]: fixed

File: concepts/05 Widgets/TagBox/07 Grouping/05 Customize Group Headers.md (hunk 2 above)
Comment (ksercs) [reply concepts/05 Widgets/TagBox/07 Grouping/05 Customize Group Headers.md]: fixed

File: concepts/05 Widgets/TagBox/02 Data Binding/05 Simple Array/10 ArrayStore.md (hunk 3 above)
Comment (ksercs) [reply concepts/05 Widgets/TagBox/02 Data Binding/05 Simple Array/10 ArrayStore.md:249]: fixed

**Review by AlexJameson — APPROVED** [review]

File: concepts/05 Widgets/TagBox/12 Limit the Tag Count/10 Selected Tags.md [hunk 4]
```diff
@@ -86,5 +86,141 @@ The following code shows the [onValueChanged](/api-reference/10%20UI%20Widgets/d
...
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+
+            this.state = {
+                isTooltipVisible: false
+            }
+
+            this.maxItems = 2;
```
Comment (ksercs) [inline concepts/05 Widgets/TagBox/12 Limit the Tag Count/10 Selected Tags.md:179]: Should we declare it as a constant outside of the class?

Comment (RomanTsukanov) [reply concepts/05 Widgets/TagBox/12 Limit the Tag Count/10 Selected Tags.md:179]: Not necessarily.

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**